spec/acceptance/nodesets/default.yml

```diff
@@ -56,6 +56,9 @@ CONFIG:
   log_level: verbose
   type:      aio
   vagrant_memsize: 256
+  ssh:
+    keepalive: true
+    keepalive_interval: 1
```

Comment:
```suggestion
  ssh:
    keepalive: true
    keepalive_interval: 1
    host_key:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:host_key].join("\n      - ") %>
    kex:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:kex].join("\n      - ") %>
    encryption:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:encryption].join("\n      - ") %>
    hmac:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:hmac].join("\n      - ") %>
```

Reply:
Acceptance tests are failing at random places with ssh connection failures for a reason unrelated to simp cli.  Should be addressed in a separate ticket.